tests/test_objects.py

```diff
@@ -32,7 +32,14 @@ def test_Product_price_setter(capsys, Product_Qq):
 
     assert message.out.strip() == "Цена не должна быть нулевая или отрицательная"
 
-    # new_product() НЕ проверяем
+
+def test_Product_add(Product_Qq):
```

Comment:
Названия функций называем в snake_Case
```suggestion
def test_product_add(Product_Qq):
```